tests/loading_data_test.py

```diff
@@ -450,3 +451,36 @@ def test_genfaces():
     assert len(data) == len(gen_faces)
 
     assert data.x["filename"].iloc[0] == "5e011b2e7b1b30000702aa59.jpg"
+
+
+def test_expand_s():
+    """Test expanding s."""
+    data = Dataset(
+        name="test",
+        filename_or_path="non-existent",
+        features=[],
+        cont_features=[],
+        sens_attr_spec={
+            "Gender": LabelSpec(["Female", "Male"], multiplier=3),
```

Comment:
Is there a good way to determine the multiplier?

Reply:
Just trying to work out if there's a way to hide this detail from the user. It might not be possible

Reply:
this can definitely be automated, but I'm hesitant to do so at this point. the biggest problem with the previous solution was too much automation, which made it very inflexible. so I'm a bit scared of automation now.

Reply:
fair enough. I like the idea of 2 levels working together, such as a higher level where things are more automated e.g. `run_suite`, and lower level where things are actually done

Reply:
but not saying such a thing needs to (or could) exist for this!

Reply:
hm, I guess there could just be a function that covers the most common use cases of this... ok I'll add it

Reply:
done